### ISSUE TITLE
Use English in code

### ISSUE THREAD

**Issue by impredicative** [AlanLuSun/High-quality-ellipse-detection#3]
Portions of the code are unreadable because they're not in English. Please update the code to be in English only, so it can be better understood by the international community. This is a reasonable expectation.

**Comment by AlanLuSun** [comment]
Yes, I will update the comments to English soon. Thanks to your reminder~But hopefully, you can call the high-quality ellipse detection function according to the instructions that I wrote.

**Comment by AlanLuSun** [comment]
Recently, for the time urgency, the main interface of high-quality ellipse detection, namely "LCS_ellipse.m", is in English. Yes, I will update the code comments of other source files to English soon. Thanks to your reminder~But hopefully, you can call the high-quality ellipse detection function according to the instructions that I wrote.
